glyphs-reader/src/font.rs

```diff
@@ -2683,6 +2725,46 @@ mod tests {
         assert_eq!(1, font.default_master_idx);
     }
 
+    #[rstest]
+    #[case::base_style_without_regular(
+        &[
+            "Expanded Thin Italic",
+            "Expanded Italic",
+            "Expanded Bold Italic",
+        ],
+        1
+    )]
+    #[case::base_style_contains_regular(
+        &[
+            "Regular Foo Bar",
+            "Regular Foo Baz",
+            "Regular Foo",
+        ],
+        2
+    )]
+    #[case::base_style_with_regular_omitted(
+        &[
+            "Condensed Thin",
+            "Condensed Light",
+            "Condensed Regular",
+        ],
+        2
+    )]
+    #[case::default_to_regular(&["Thin", "Light", "Regular", "Medium", "Bold"], 2)]
+    #[case::default_to_first(&["Foo", "Bar", "Baz"], 0)]
+    fn find_default_master(#[case] master_names: &[&str], #[case] expected_idx: usize) {
```

Comment:
Optional: the cases would be easier to read if the expected answer was also a string

Reply:
I would also suggest a common on each for why, e.g. 

```rust
    #[case::base_style_without_regular(
        &[
            "Expanded Thin Italic",
            "Expanded Italic",
            "Expanded Bold Italic",
        ],
        1  // common words "Expanded Italic", exact match at [1]
    )]
    #[case::base_style_contains_regular(
        &[
            "Regular Foo Bar",
            "Regular Foo Baz",
            "Regular Foo",
        ],
        2. // common words "Regular Foo", exact match at [2]
    )]
    #[case::base_style_with_regular_omitted(
        &[
            "Condensed Thin",
            "Condensed Light",
            "Condensed Regular",
        ],
        2 // common words "Condensed", matches [2] when "Regular" dropped
    )]
    // No common words, "Regular" wins
    #[case::default_to_regular(&["Thin", "Light", "Regular", "Medium", "Bold"], 2)]
   // No common words, no "Regular", first entry wins
    #[case::default_to_first(&["Foo", "Bar", "Baz"], 0)]
```

Reply:
> the cases would be easier to read if the expected answer was also a string

done in 4dcde7ac